### PR TITLE
Addition of Chronicle Detection Rules

### PR DESCRIPTION
**Pull request by R1shal1n**
New Chronicle detection rules added:

- 1_01_login_highly_priviledged_account.yaral		
- 1_02_suspicious_login_attempt.yaral
- 2_01_super_admin_or_admin_permissions_granted.yaral				
- 2_10_org_admin_permissions_granted.yaral
- 3_14_dns_zones_modified_or_deleted.yaral		
- 6_01_hosts_reaching_to_many_other_hosts_or_ports.yaral

### CODE REVIEW

File: yaral/2_10_org_admin_permissions_granted.yaral [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  meta:
+    author = "Google Cloud Security"
+    description = "A user is granted an admin privilege"
+    severity = "HIGH"
+    url = "https://developers.google.com/admin-sdk/reports/v1/appendix/activity/admin-delegated-admin-settings"
+
+  events:
+    $workspace.metadata.vendor_name        = "Google Workspace"
+    $workspace.metadata.product_name       = /admin/ nocase
+    $workspace.metadata.product_event_type = /assign_role/ nocase
```
Comment (rarsan) [inline yaral/2_10_org_admin_permissions_granted.yaral:27]: This looks equivalent to above `yaral/2_01_super_admin_or_admin_permissions_granted.yaral` rule.
2.10 use case should trigger on setIAMPolicy event, not assignRole which is slightly different. (see equivalent [2.10 SQL](https://github.com/GoogleCloudPlatform/threat-detection-as-code/blob/main/sql/2_10_org_admin_permissions_granted.sql#L24))

**Comment by rarsan** [comment]
Thanks for the contribution! LGTM. One question below pertaining to 2.10 rule.

**Review by rarsan — CHANGES_REQUESTED** [review]
Thanks for the contribution! LGTM. Just one requested change pertaining to 2.10 rule.

File: yaral/2_10_org_admin_permissions_granted.yaral (hunk 1 above)
Comment (R1shal1n) [reply yaral/2_10_org_admin_permissions_granted.yaral]: Thanks, I have corrected this

**Pull request merged**